src/devices/Adxl345/Adxl345.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+using System.Runtime.InteropServices;
+using System.Device.Spi.Drivers;
+using System.Numerics;
+
+namespace Iot.Device.Adxl345
+{
+    /// <summary>
+    /// SPI Accelerometer ADX1345
+    /// </summary>
+    public class Adxl345 : IDisposable
+    {
+        private SpiDevice _sensor = null;
+
+        private readonly int _busId;
+        private readonly int _chipSelect;                                
+        private readonly byte _gravityRangeByte;                             
+        private readonly int _range;
+
+        private const int RESOLUTION = 1024;
```

Comment:
Unless you are trying to match exactly what the data sheet says (which doesn't seem to be the case here, you should use CammelCase naming for constants like: `Resolution`